libheif/heif.h

```diff
@@ -1934,6 +1935,11 @@ struct heif_error heif_region_get_polyline_points_scaled(const struct heif_regio
                                                          double* pts,
                                                          heif_item_id image_id);
 
+LIBHEIF_API
+struct heif_error heif_region_get_referenced_mask(const struct heif_region* region,
+                                                  int32_t* x, int32_t* y,
+                                                  uint32_t* width, uint32_t* height);
+
```

Comment:
I think this should also return the ID of the referenced mask image. If I understand the standard correctly, we would get the ID from the 'iref' box.
Do we have an example image?

Reply:
I do not have a test image, and am working on creating some.

Will extend the API and implementation to provide the item id.

Reply:
I have added the API and implementation, along with special case handling for when width and height are 0 in the property (which flags that we need to get the values from the referenced item `ispe`).

Still need to make representative test data.